Apidaze.SDK.Tests.Integration/IvrExample/Program.cs

```diff
@@ -0,0 +1,326 @@
+
+        /// <summary>
+        /// Call this to start the web server
+        /// </summary>
+        public static void StartWebServer()
+        {
+            if (_mainLoop != null && !_mainLoop.IsCompleted) return; //Already started
+            _mainLoop = MainLoop();
+        }
+
+        /// <summary>
+        /// Call this to stop the web server. It will not kill any requests currently being processed.
+        /// </summary>
+        public static void StopWebServer()
+        {
+            _keepGoing = false;
+            lock (Listener)
+            {
+                Listener.Stop();
+            }
+            try
+            {
+                _mainLoop.Wait();
+            }
+            catch { }
+        }
+
+        /// <summary>
+        /// The main loop to handle requests into the HttpListener
+        /// </summary>
+        /// <returns></returns>
+        private static async Task MainLoop()
+        {
+
+            Listener.Prefixes.Add(LOCALHOST + INTRO);
+            Listener.Prefixes.Add(LOCALHOST + STEP_1_PATH);
+            Listener.Prefixes.Add(LOCALHOST + STEP_2_PATH);
+            Listener.Prefixes.Add(LOCALHOST + STEP_3_PATH);
+            Listener.Prefixes.Add(LOCALHOST + PLAYBACK_PATH);
+
+            Listener.Start();
+            while (_keepGoing)
+            {
+                try
+                {
+                    var context = await Listener.GetContextAsync();
+                    lock (Listener)
+                    {
+                        if (_keepGoing) ProcessRequest(context);
+                    }
+                }
+                catch (Exception e)
+                {
+                    Console.WriteLine(e.Message);
+                    if (e is HttpListenerException) return;
+                }
+            }
+        }
+
+        /// <summary>
+        /// Handle an incoming request
+        /// </summary>
+        /// <param name="context">The context of the incoming request</param>
+        private static void ProcessRequest(HttpListenerContext context)
+        {
+            using var response = context.Response;
+            try
+            {
+                var handled = false;
+                switch (context.Request.Url.AbsolutePath)
+                {
+                    case "/":
+                        handled = GetIntro(context, response, handled);
+                        break;
+                    case "/step1.xml":
+                        handled = GetStep1(context, response, handled);
+                        break;
+                    case "/step2.xml":
+                        handled = GetStep2(context, response, handled);
+                        break;
+                    case "/step3.xml":
+                        handled = GetStep3(context, response, handled);
+                        break;
+                    case "/Apidazeintro.wav":
+                        handled = GetIntroWav(context, response, handled);
+                        break;
```

Comment:
Passing `handled` to these functions are unnecessary, in my opinion. Those functions respond `true`/`false` regardless of it.